src/content/en/2021/resource-hints.md

```diff
@@ -1,20 +1,609 @@
+
+#### `use-credentials`
+
+When requesting cross-origin resources which require authentication, for example through the use of cookies, client certificates or the `Authorization` header; setting the `crossorigin="use-credentials"` attribute will include this data in the request and allowing the server to respond to the request so that the resource may be preloaded. This is not a common scenario with 0.1% usage, however if your page content is dependent on an authenticated status, it could be used to initiate an early fetch request to get the login status.
+
+### The `media` attribute
+
+An oft-neglected feature available to `rel="preload"`, is the ability to specify media queries through the `media` attribute—with less than 4% of all preloads using this attribute. The `media` attribute accepts media queries allowing you to target the media type and specific browser features, such as viewport width. As an example, the `media` attribute would allow you to preload a low-resolution image on devices with a narrow viewport and a full-sized image on devices with a large viewport.
+
+In addition to the `media` attribute, the `<link>` element supports `imagesrcset` and `imagesizes` attributes which correspond to the `srcset` and `sizes` attributes on `<img>` elements. Using these attributes, you can use the same resource selection criteria that you would use on your image. Unfortunately, their adoption is very low (less than 1%); most likely owing to the lack of <a hreflang="en" href="https://caniuse.com/mdn-html_elements_link_imagesizes">support</a> on Safari.
+
+<p class="note">Note: The `media` attribute is not available on all `<link>` elements as the spec suggests, but it is only available on `rel="preload"`.</p>
+
+### Bad practices
+
+Owing to the versatility of `rel="preload"`, there isn't a clear set of rules dictating how to implement the preload hint, but we can learn a lot from our mistakes and understand how to avoid them.
+
+#### Unused preload
+
+We have already seen that there is a negative correlation between a website's performance and the number of preload hints. This relationship may be influenced by two factors:
+- Incorrect preloads
+- Unused preloads
+
+##### Incorrect preloads
+
+When you preload a resource which is not as important as the other resources which the browser would have otherwise prioritized. We are unable to measure the extent of incorrect preloads as you would need to A/B test the page with and without each hint.
+
+##### Unused preloads
```

Comment:
It feels strange that "Unused preloads" is a subsection within "Unused preload". Could we remove these subheadings and nest the content directly under "Unused preload"? May also need to rephrase each section to clarify which bad practice it's referring to.

Reply:
Rephrased section.